src/hotspot/share/opto/macroArrayCopy.cpp

```diff
@@ -1095,8 +1097,14 @@ MergeMemNode* PhaseMacroExpand::generate_slow_arraycopy(ArrayCopyNode *ac,
   }
   transform_later(out_mem);
 
-  *io = _callprojs.fallthrough_ioproj->clone();
-  transform_later(*io);
+  // When src is negative and arraycopy is before an infinite loop,_callprojs.fallthrough_ioproj
+  // could be NULL. Not clone and update NULL fallthrough_ioproj.
```

Comment:
Maybe replace "Not clone" by "Skip clone".

Reply:
Thanks! Updated.